src/main/java/org/skriptlang/skript/bukkit/event/EventConverter.java

```diff
@@ -0,0 +1,23 @@
+package org.skriptlang.skript.bukkit.event;
```

Comment:
i kind of changed my mind on this again. if we happen to rework event value registration, new classes would go here. since this is part of the "old"/current system, i think it'd be fine to just go at `ch.njol.skript.registrations` or even as a class within `EventValues`

Reply:
Moved again 👍